src/Compilers/CSharp/Test/Semantic/Semantics/DeconstructionTests.cs

```diff
@@ -822,6 +1257,54 @@ static void Main()
             CompileAndVerify(comp, expectedOutput: "1 2");
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TupleWithUseSiteError_IOperation()
```

Comment:
What is the use-site error?

Reply:
This is a clone of the above test `TupleWithUseSiteError`, and yes there doesn't seem to be any use-site diagnostic in that test as well. I will rename both the tests.